### DISCUSSION TITLE
[Resolved] Can't login to vercel

### DISCUSSION THREAD

**Discussion by sgarza** [vercel/vercel#7007] [Help] [answered]
I sued to login with Github oauth but its not working I got "There is no account linked error"

When I try to signup with email I get the link in my email, click it and I get: 

```
Email Address Confirmed

You have been authenticated successfully and are now logged in…
```
but it didn't logme in

**Comment by kyds3k** [comment]
I have the same problem now, did you ever get it fixed?

**Reply by amyegan** [reply, marked as answer]
Hi @kyds3k. Please send us an email at [registration@vercel.com](mailto:registration@vercel.com) with the email address or username you used to sign up, and the team will help get this solved for you.

**Reply by yoharden** [reply]
I have the same question too

**Comment by httpsmarks** [comment]
I have the same problem, Is there a way to fix?

**Reply by amyegan** [reply]
Sorry you're running into this error, @httpsmarks. I think you're being affected by this incident: https://www.vercel-status.com/incidents/590pcktngq7z

It looks like the situation has improved now, but you can follow along there for the latest updates.
